.github/workflows/weekly-cea.yml

```diff
@@ -0,0 +1,51 @@
+name: Weekly CEA CI
```

Comment:
```suggestion
name: Weekly CEA builds
```
Technically this is not Continuous Integration :)

Reply:
Right!